vars/tdr.groovy

```diff
@@ -13,6 +13,16 @@ def runEndToEndTests(int delaySeconds, String stage, String buildUrl) {
     wait: false)
 }
 
+def runGitSecrets(repo) {
+  sh "set +e"
+  def exitCode = sh(script: "git-secrets --scan", returnStatus: true)
+  sh "set -e"
+  if(exitCode == 1) {
```

Comment:
Is it worth catching any non-zero exit code?

Reply:
Yeah I think so. I've changed it.